Dockerfile

```diff
@@ -1,10 +1,10 @@
-FROM hypriot/rpi-alpine:3.4
+FROM armhf/alpine:3.4
```

Comment:
@nsteinmetz Can you change that line back to `hypriot/rpi-alpine:3.4` to see how it works in Travis?

See also the latest blog post https://blog.hypriot.com/post/setup-simple-ci-pipeline-for-arm-images/